test/js/sql/sqlite-sql.test.ts

```diff
@@ -1311,6 +1311,130 @@ describe("SQL helpers", () => {
     expect(results[0].value).toBe("test");
   });
 
+  test("insert select helper with IN", async () => {
+    const random_name = "test_" + randomUUIDv7("hex").replaceAll("-", "");
```

Comment:
I don't think this is a great idea. I'm usually guilty of this junk myself, but usually randomness in tests which isn't guided in any shape or form leads to flaky non-reproducible tests. (I say that as I've done the same in `Redis`).

I would honestly use a constant here or if you think there is some impact we would get by fuzzing the input, then perhaps we can figure out how to do that.